libraries/AP_HAL_ChibiOS/CANFDIface.cpp

```diff
@@ -304,24 +304,34 @@ bool CANIface::computeTimings(const uint32_t target_bitrate, Timings& out_timing
      *
      */
     if ((target_bitrate != (pclk / (prescaler * (1 + solution.bs1 + solution.bs2)))) || !solution.isValid()) {
+        Debug("Timings: Invalid Solution %lu %lu %d %d %lu \n", pclk, prescaler, int(solution.bs1), int(solution.bs2), (pclk / (prescaler * (1 + solution.bs1 + solution.bs2))));
         return false;
     }
 
-    Debug("Timings: quanta/bit: %d, sample point location: %.1f%%\n",
-          int(1 + solution.bs1 + solution.bs2), float(solution.sample_point_permill) / 10.F);
-
     out_timings.prescaler = uint16_t(prescaler - 1U);
     out_timings.sjw = 0;                                        // Which means one
     out_timings.bs1 = uint8_t(solution.bs1 - 1);
     out_timings.bs2 = uint8_t(solution.bs2 - 1);
     return true;
 }
 
+static uint32_t bytes_to_word(const uint8_t data[]) {
```

Comment:
unused fn

Reply:
it was rendered unused because of merge issue, its being used now.